policies/pipeline/basic.rego

```diff
@@ -0,0 +1,22 @@
+package policies.pipeline.basic
+
+import data.lib
+
+expected_kind := "Pipeline"
+
+# (Not sure if we need this, but I'm using it to test the docs build.)
+
+# Fixme: It doesn't fail if the kind key is entirely missing..
+
+# METADATA
+# title: Check the kind is "PipelineRun"
```

Comment:
```suggestion
# title: Check the kind is "Pipeline"
```

Reply:
+1 on this suggested change, see associated comment in `index.md`